### PR TITLE
AIR-1007: Add debug messages for LIDAR version and distance data

### PR DESCRIPTION
**Pull request by dat-tran-matternet**

### CODE REVIEW

File: libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp [hunk 1]
```diff
@@ -467,21 +467,45 @@ void AP_RangeFinder_LightWareI2C::update(void)
 
 void AP_RangeFinder_LightWareI2C::legacy_timer(void)
 {
+    static const uint32_t TIMER_FREQ = 20;
```
Comment (viet-nguyen-matternet) [inline libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp:470]: Is there a reason to not reduce the period of the timer itself and instead use this static counter?

File: libraries/AP_Parachute/AP_Parachute.cpp [hunk 2]
```diff
@@ -283,6 +283,8 @@ void AP_Parachute::mttr_fts_update()
                 memset(_mttr_fts_version, 0, sizeof(_mttr_fts_version));
                 memcpy(_mttr_fts_version, msg->git_hash, sizeof(msg->git_hash));
                 AP::logger().Write("FTSV", "TimeUS,Hash", "QN", AP_HAL::micros64(), _mttr_fts_version);
+                // Manufacturing test debug message
+                gcs().send_text(MAV_SEVERITY_INFO, "FTS version: %s", _mttr_fts_version);
```
Comment (viet-nguyen-matternet) [inline libraries/AP_Parachute/AP_Parachute.cpp:287]: meant to be part of this PR?

**Review by viet-nguyen-matternet — APPROVED** [review]

File: libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp (hunk 1 above)
Comment (dat-tran-matternet) [reply libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp:470]: This is a hack to add debug data. You don't want to change the flow of the original function.

File: libraries/AP_Parachute/AP_Parachute.cpp (hunk 2 above)
Comment (dat-tran-matternet) [reply libraries/AP_Parachute/AP_Parachute.cpp:287]: It's for manufacturing, not production.

**Pull request merged**